Add ServiceCatalog subchart with crds (#3601)
kyma-project/kyma: docs/service-catalog/docs/03-04-experimental-features.md

---
title: Experimental features
type: Details
---

> Currently Service Catalog requires its own instance of api-server and etcd. This adds additional complexity to the cluster configuration and increases

Reviewer: besides complexity and configuration please also mention that in case of APi-server downtime, all of the SC resources are not available (and from time to time it happens, simple pod restart can result with such behaviour)

maintenance costs. Kyma developers are contributing to the Service Catalog project to remove dependency to these external components and replace them 
with native K8S solution - Custom Resource Definition.

### Enable CRDs

To enable the CRDs feature you have to override parameters `service-catalog-apiserver.enabled` and `service-catalog-crds.enabled`

> in the installer-config file. Modify or add the `service-catalog-overrides` config map:  

Reviewer: Mention exactly which file to modify(link to local config) and in which one to add a CM (link to cluster cluster)

---
title: Experimental features
type: Details
---

Currently Service Catalog requires its own instance of api-server and etcd. This adds additional complexity to the cluster configuration and increases

> maintenance costs. In case of api-server downtime all Service Catalog resources are not available (even simple pod restart can result in such a behaviour).

Reviewer 2: ```suggestion
In case of api-server downtime, all Service Catalog resources are unavailable. What is more, even the simple Pod restart can result in such behavior.
```

Kyma developers are contributing to the Service Catalog project to remove dependency to these external components and replace them 
with native K8S solution - Custom Resource Definition.

### Enable CRDs

To enable the CRDs feature you have to override parameters `service-catalog-apiserver.enabled` and `service-catalog-crds.enabled`
in the installer-config file:

> - for a local installation modify the `service-catalog-overrides` config map in [installer-config-local.yaml](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-local.yaml.tpl#L73):

Reviewer 2: ```suggestion
- For the local installation, modify the `service-catalog-overrides` ConfigMap in the [installer-config-local.yaml](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-local.yaml.tpl#L73) file:
```
